src/category_theory/concrete_category/basic.lean

```diff
@@ -42,6 +42,9 @@ class concrete_category (C : Type (u+1)) extends category.{u} C :=
 (forget : C ⥤ Type u)
 [forget_faithful : faithful forget]
 
+instance (C : Type (u+1)) [concrete_category C] : has_coe_to_sort C :=
```

Comment:
This is probably a bad instance, since it matches virtually every type. Can we do without it?

Reply:
Ah. This is a good point, but also makes me sad. :-) We'll have to add a corresponding line to every concrete category.

Reply:
Okay! I think I found a good solution, adding an instance indexed on `induced_category`, that covers nearly all of our use cases so far.